examples/v1beta1/nodeclaim/manual-nodeclaim.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: karpenter.sh/v1beta1
+kind: NodeClaim
+metadata:
+  name: manual-kaito
+  labels:
+    karpenter.sh/nodepool: kaito
+  annotations:
+    karpenter.sh/do-not-disrupt: "true"
+spec:
+  nodeClassRef:
+    name: default
+    kind: AKSNodeClass
+  taints:
+    - key: "sku"
+      value: "gpu"
+      effect: NoSchedule
+  requirements:
+    - key: "karpenter.sh/nodepool"
+      operator: In
+      values: ["kaito"]
```

Comment:
Can you explain why we need this?

Reply:
My intention in leaving this would be to make testing easier in the future. I can go ahead and remove it for now. If, in the future, we decide to add testing around manual nodeclaim creation, maybe this is worthwhile using. TBD + can access again from the commit here.